test.js

```diff
@@ -11,7 +13,7 @@ test('Can generate and decode a signing request', async t => {
   const toSign = await generate(core)
   const decoded = decode(toSign)
 
-  t.is(decoded.version, 1, 'Current version is 1')
+  t.is(decoded.version, 2, 'Current version is 1')
```

Comment:
Typo: should say `'Current version is 2'` (or future proof `'Current version is correct'`)